### DISCUSSION TITLE
client.channelList 에서 플러스친구 대화방 채널을 가져오는 방법은 없을까요?

### DISCUSSION THREAD

**Discussion by dlehdanakf** [storycraft/node-kakao#261] [Q&A] [answered]
채널 리스트를 호출할 경우 일반 대화방, 오픈채팅 대화방 목록만 반환되던데, 플러스친구와 대화했던 채널의 ID를 가져오는 방법이 있을지 궁금하여 문의드립니다~

감사합니다.

**Comment by storycraft** [comment, marked as answer]
플러스친구의 채팅방은 일반대화방 목록에 있을 것으로 보입니다. `TalkChannel`객체의 `info`객체 `type`속성을 이용해서 구별해 주세요.

**Reply by storycraft** [reply]
알려진 채팅방 타입은 이곳에서 찾을 수 있습니다. https://github.com/storycraft/node-kakao/blob/51960f48f23d05134b748cf873f09fdbc7362831/src/channel/channel-type.ts#L7
